### PR TITLE
Manual Brake - Further update

### PR DESCRIPTION
**Pull request by peternewell**
https://blueprints.launchpad.net/or/+spec/manual-braking - manual brake updates.

Added additional manual information.

### CODE REVIEW

File: Source/Documentation/Manual/physics.rst [hunk 1]
```diff
@@ -2246,6 +2246,9 @@ The brake can be applied by pressing ], and released by pressing the [ keys.
 To control the application and release rates on the brake use the ``EngineBrakesControllerMaxApplicationRate`` and 
 ``EngineBrakesControllerMaxReleaseRate`` parameters.
 
+A steam effect can be added by adding the following special effects to the wagon ``SteamBrakeFX``. This special effect will 
+turn on and off with the brake operation and can be used to model steam leakage of the steam brake cylinder, etc.
```
Comment (YoRyan) [inline Source/Documentation/Manual/physics.rst:2250]:
```suggestion
The ``SteamBrakeFX`` special effect, if added to the wagon, will turn on and 
off with the brake operation and can be used to model steam leakage of the 
steam brake cylinder, etc.
```
I suggest rewording this to flow more naturally.

**Review by YoRyan — COMMENTED** [review]
Hi Peter,

Apologies for the long delay. Thanks for remembering to include documentation.

File: Source/Documentation/Manual/physics.rst (hunk 1 above)
Comment (peternewell) [reply Source/Documentation/Manual/physics.rst]: Suggested change added.

**Review by YoRyan — APPROVED** [review]
